vars/dockerBuildArgs.groovy

```diff
@@ -114,7 +114,7 @@ String call(Map config = [:]) {
                      " --build-arg JENKINS_URL=$env.JENKINS_URL"
     if (cachebust) {
       Calendar current_time = Calendar.getInstance()
-      ret_str += " --build-arg CACHEBUST=${currentBuild.startTimeInMillis}"
+      ret_str += " --build-arg CACHEBUST=" + current_time.get(Calendar.WEEK_OF_YEAR)
```

Comment:
(Realizing that I +1'd this,) this really should not have been changed here.  `CACHEBUST` was (is) supposed to always guarantee that the remaining portion of a `Dockerfile` will be executed.  While it was being used to force updates, that was not it's specific goal.  There are other use-cases for wanting to force execution in a `Dockerfile`.

If we wanted to create a new trigger/timer for doing updates, a new variable should have been created.

We probably want a number of `CACHEBUST` variables at various granularities named accordingly.  I.e. CB_WEEKLY.